Vagrantfile

```diff
@@ -9,7 +9,8 @@ Vagrant.configure("2") do |config|
     override.vm.hostname = "quadlet.example.com"
 
     override.vm.provider "libvirt" do |libvirt, provider|
-      libvirt.memory = "4096"
```

Comment:
Was nice while it lasted!

Reply:
Yeah, and produced nice random errors when someone tried to do any two trivial actions with candlepin in a short timeframe ;)